### PR TITLE
Create release.yml

### PR DESCRIPTION
**Pull request by swoolcock**

### CODE REVIEW

**Comment by LumpBloom7** [comment]
Just to note, using the default github token for releasing will create a release under the name of "github-actions", instead of yourself.

For other actions the github token is sufficient.

**Comment by LumpBloom7** [comment]
Also the changelog generation template is `.grenrc` (other formats also accepted), so you may want to set that up beforehand. 

Some of the template options doesn't work in `gren Release` unfortunately in my case, so some things I copy and paste in after the release is made.

**Comment by swoolcock** [comment]
Ah ok, thanks.  Yeah I'm new to GitHub actions, I've only worked with Jenkins and AppVeyor in the past.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        tag_name: ${{ github.ref }}
+        release_name: ${{ github.ref }}
+    - name: Upload Release Asset
+      id: upload-release-asset
+      uses: actions/upload-release-asset@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
+        asset_path: ./osu.Game.Rulesets.Rush/bin/Release/netcoreapp3.1/osu.Game.Rulesets.Rush.dll
```
Comment (LumpBloom7) [inline .github/workflows/release.yml:39]: Oh, just noticed, but you should change `netcoreapp3.1` to `netstandard2.1` as your main project doesn't output to that folder.

Comment (swoolcock) [reply .github/workflows/release.yml:39]: Thanks, I'll fix it up tomorrow.

**Pull request merged**